static/scss/partials/main.scss

```diff
@@ -1575,17 +1576,19 @@ input.input-has-error {
   left: 0;
   right: 0;
   pointer-events: none;
-  background: #2f2f54;
+  background: $color-white;
```

Comment:
Let's just focus on the text link change for now. Can you revert all color updates? Let's update the UI all at once, when it's locked in with Eduardo.